### ISSUE TITLE
config superset for local time zone?

### ISSUE THREAD

**Issue by Manoj1881** [apache/superset#6768]
Make sure these boxes are checked before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same open issue and I haven't found any solution.

### Superset version
superset --version
Flask 0.12.4
Python 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 12:04:33)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]

### Expected results
querying from superset is still in UTC even after changing DRUID_TZ in  config.py to DRUID_TZ = tz.tzlocal()

**Comment by sivasyam1881** [comment]
Thank you for the response, the hour offset is taking only integers, I need another 30 min offset.
I am going through the code to see where I can add that.

**Comment by sai1881** [comment]
Thank you for the response, the hour offset is taking only integers, I need another 30 min offset.
I am going through the code to see where I can add that.

**Comment by Manoj1881** [comment]
Thanks for the reply @srggrs 
I am in IST with 5 hours and 30 min offset and hour offset is only accepting integers,

I think it should accept float value.

or
the timestamp should be converted back to local.

**Comment by TheRum** [comment]
@Manoj1881 I am also having same issue. Did you find any workaround to this ?

**Comment by med4it** [comment]
Same issue, is there any way to change timezone in superset from the UI ?

**Comment by krishmathakral** [comment]
@Manoj1881 @srggrs 

Can you please suggest any workaround for changing timezone from UTC to IST

**Comment by krishmathakral** [comment]
@Manoj1881 @srggrs

Can you please suggest any workaround for changing timezone from UTC to IST

**Comment by jpike88** [comment]
+1 on this, timezones are important.

**Comment by D3nn3** [comment]
Agree. We have some edge cases were entities are associated to wrong days (leading to sometimes inaccurate metrics) because of this :/

**Comment by jessicaxyl** [comment]
+1

**Comment by zygeilit** [comment]
+1

**Comment by zygeilit** [comment]
timezones are important

**Comment by DevHerrera** [comment]
+1

**Comment by etothexipi** [comment]
+1 Daylight savings causes issues where users have to update datasource timezone offset for each datasource twice per year.

**Comment by yeongjukang** [comment]
+1
